zipkin-autoconfigure/collector-kafka08/README.md

```diff
@@ -13,14 +13,14 @@ JRE 8 is required to run Zipkin server.
 Fetch the latest released
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
```

Comment:
```suggestion
[executable jar for Zipkin server](https://search.maven.org/remote_content?g=org.apache.zipkin.zipkin2&a=zipkin-server&v=LATEST&c=exec)
```

Reply:
there's a separate task for this, as this isn't published yet. #2513